client/src/ReviewModule.jsx

```diff
@@ -24,6 +24,7 @@ const ReviewModule = (props) => {
 
     axios.get(`/api/reviews/products/${productId}`).then((response) => {
       setReviews(response.data);
+      console.log(response.data);
```

Comment:
@dbooyah93 all I see is a console.log line added. If that's all you intended to do then you're good to go but I just wanted to confirm.